tests/ci/aml_tests_github_actions/test_groups.py

```diff
@@ -0,0 +1,285 @@
+        "tests/unit/examples/test_notebooks_gpu.py::test_gpu_vm",  # 0.76s
+        "tests/smoke/recommenders/recommender/test_newsrec_model.py::test_model_npa",  # 202.61s
+        "tests/smoke/examples/test_notebooks_gpu.py::test_npa_smoke",  # 366.22s
+        "tests/integration/examples/test_notebooks_gpu.py::test_npa_quickstart_integration",  # 810.92s
+        #
+        "tests/smoke/examples/test_notebooks_gpu.py::test_wide_deep_smoke",  # 122.71s
+    ],
+    "group_gpu_006": [  # Total group time: 1844.05s
+        "tests/unit/examples/test_notebooks_gpu.py::test_gpu_vm",  # 0.76s
+        "tests/integration/examples/test_notebooks_gpu.py::test_wide_deep_integration",  # 1843.29s
+    ],
+    "group_gpu_007": [  # Total group time: 1278.96s
+        "tests/unit/examples/test_notebooks_gpu.py::test_gpu_vm",  # 0.76s
+        "tests/smoke/recommenders/recommender/test_newsrec_model.py::test_model_nrms",  # 188.60s
+        "tests/smoke/examples/test_notebooks_gpu.py::test_nrms_smoke",  # 232.55s
+        "tests/integration/examples/test_notebooks_gpu.py::test_nrms_quickstart_integration",  # 857.05s
+    ],
+    "group_gpu_008": [  # Total group time: 1208.62s
+        "tests/unit/examples/test_notebooks_gpu.py::test_gpu_vm",  # 0.76s
+        "tests/smoke/examples/test_notebooks_gpu.py::test_lstur_smoke",  # 246.46s
+        "tests/smoke/recommenders/recommender/test_newsrec_model.py::test_model_lstur",  # 194.88s
+        "tests/integration/examples/test_notebooks_gpu.py::test_lstur_quickstart_integration",  # 766.52s
+    ],
+    "group_gpu_009": [  # Total group time: 1063.51s
+        "tests/unit/examples/test_notebooks_gpu.py::test_gpu_vm",  # 0.76s
+
+        #FIXME: https://github.com/microsoft/recommenders/issues/1716
+        # "tests/integration/examples/test_notebooks_gpu.py::test_sasrec_quickstart_integration",  # 448.06s + 614.69s
+    ],
+    "group_gpu_010": [  # Total group time: 2034.61s
+        "tests/unit/examples/test_notebooks_gpu.py::test_gpu_vm",  # 0.76s
+        "tests/integration/examples/test_notebooks_gpu.py::test_naml_quickstart_integration",  # 2033.85s
+    ],
+    "group_spark_001": [  # Total group time: 845.16s
+        "tests/smoke/recommenders/dataset/test_movielens.py::test_load_spark_df",  # 4.33s
+        "tests/integration/recommenders/datasets/test_movielens.py::test_load_spark_df",  # 25.58s + 101.99s + 139.23s
+        #
+        "tests/smoke/recommenders/dataset/test_criteo.py::test_criteo_load_spark_df",  # 6.83s
+        "tests/smoke/examples/test_notebooks_pyspark.py::test_mmlspark_lightgbm_criteo_smoke",  # 32.45s
+        "tests/integration/recommenders/datasets/test_criteo.py::test_criteo_load_spark_df",  # 374.64s
+        #
+        "tests/smoke/examples/test_notebooks_pyspark.py::test_als_pyspark_smoke",  # 49.53s
+        "tests/integration/examples/test_notebooks_pyspark.py::test_als_pyspark_integration",  # 110.58s
+    ],
+}
+
+unit_test_groups = {
+    "group_pyspark_001": [
```

Comment:
can you add the timeof each group as it is done above?

Reply:
added total time for each group.